src/Chapter2.hs

```diff
@@ -336,7 +337,13 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList s e l
+    | s < 0 = []
+    | e < s = []
+    | otherwise = tr (e - s + 1) (tr (e + 1) l)
+      where
+        tr :: Int -> [a] -> [a]
+        tr t nl = reverse (take t nl)
```

Comment:
Interesting idea 🧐 

But it would use `reverse` twice, which is quite slow on lists.
Here would be better to use standard function s`drop` and `take` and their combination together can give you the result 🙂